gulpfile.babel.js

```diff
@@ -103,6 +103,7 @@ function humanize(categoryId){
   switch(categoryId){
     case 'aspnet': return 'ASP.NET';
     case 'aws': return 'AWS';
+    case 'azureFunctions': return 'Azure Functions';
```

Comment:
I added a new category in order to use the Functions logo, but this could just as easily fit within the Azure category if preferred.